source/adapters/cuda/tensor_map.cpp

```diff
@@ -0,0 +1,142 @@
+//===--------- tensor_map.cpp - CUDA Adapter ------------------------------===//
+//
+// Copyright (C) 2024 Intel Corporation
+//
+// Part of the Unified-Runtime Project, under the Apache License v2.0 with LLVM
+// Exceptions. See LICENSE.TXT
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include <cuda.h>
+#include <ur_api.h>
+
+#include "context.hpp"
+
+struct ur_exp_tensor_map_handle_t_ {
+  CUtensorMap Map;
+};
+
+#define CONVERT(URTYPE, CUTYPE)                                                \
+  if (URTYPE & UrType)                                                         \
```

Comment:
Might be prudent to have `(URTYPE)` and `(CUTYPE)` just to avoid any surprises.

Reply:
Good idea